### ISSUE TITLE
[BUG] initialScrollIndex "-1" is not valid

### ISSUE THREAD

**Issue by efstathiosntonas** [MobileReality/react-native-select-pro#214]
Hi, I'm using `2.1.5` and it throws this error when I open the drop down: 

`initialScrollIndex "-1" is not valid`

I've changed this line:

https://github.com/MobileReality/react-native-select-pro/blob/6a9ac023cf0bbb25027f466503a9a05cec003dd3/packages/react-native-select-pro/src/components/flat-options-list/flat-options-list.tsx#L28

to: 
```tsx         
initialScrollIndex={initialScrollIndex === -1 ? 0 : initialScrollIndex}
```

But I don't know yet if there are any side effects, probably there aren't any, let me know if you want me to open a PR

**To Reproduce**
Nothing special, just:

```tsx
 <Select
        clearable={false}
        flatListProps={{ indicatorStyle: theme.mode === "dark" ? "white" : "black" }}
        onSelect={setSelectedTicket}
        options={tickets || []}
        placeholderText={t("support_screen.select_box_placeholder") as string}
        placeholderTextColor={themeContext.newPost.select.placeholder}
        styles={selectStyles}
      />
```

**Expected behavior**

**Screenshots**
![Screenshot 2023-05-18 at 08 44 22](https://github.com/MobileReality/react-native-select-pro/assets/717975/a86d49f2-e63e-4831-88b3-10f5f1aa697a)

**Comment by efstathiosntonas** [comment]
Hi @gitsad, I can't find out why `initialScrollIndex` is `-1` to start with, haven't checked on source code where this is calculated so I over driven it to stop showing the error. No special use cases, check on how I'm using the module, pretty simple and straight forward usage.

**Comment by efstathiosntonas** [comment]
Now that I'm thinking of it, I believe it doesn't like the empty array: ` options={tickets || []}` thus it cannot find the `initialScrollIndex`.

I have to either wait for tickets to load or patch it as mentioned above.

**Comment by efstathiosntonas** [comment]
closing this, fixed on rn `0.71.9` [here](https://github.com/facebook/react-native/issues/36066)
